Install vagrant from Vagrant website instead of SCL

raghavendra-talur opened the pull request:
We have been using vagrant and vagrant-libvirt packages from CentOS SCL but we have an installation error now because of dependency issues.

While the SCL fixes the issues, we can use vagrant from the vagrant website.

phlogistonjohn commented on heketi-centos-ci-tests.sh:
```diff
@@ -22,8 +33,6 @@ yum -y install \
 	qemu-kvm-tools \
 	qemu-img \
-	sclo-vagrant1-vagrant \
-	sclo-vagrant1-vagrant-libvirt \
```
Perhaps converge this 'yum install' and the one above instead of invoking 'yum install' twice?

phlogistonjohn commented:
How is this progressing?

raghavendra-talur committed:
Use a wrapper for calling vagrant

Depending on how vagrant is installed the invocation is different.

Signed-off-by: Raghavendra Talur <rtalur@redhat.com>

raghavendra-talur committed:
use vagrant from upstream

Signed-off-by: Raghavendra Talur <rtalur@redhat.com>

raghavendra-talur committed:
install libvirt

When vagrant is installed from the Vagrant website, it does not pull the
libvirt dependencies. We also install the "Development Tools" group
because vagrant plugin installations depend on them.

Signed-off-by: Raghavendra Talur <rtalur@redhat.com>

phlogistonjohn commented on heketi-centos-ci-tests.sh:
```diff
@@ -33,7 +33,11 @@ yum -y install \
 	python-virtualenv \
-	ansible
+	ansible \
+        libvirt \
```
Indentation level changed?

phlogistonjohn commented on heketi-centos-ci-tests.sh:
```diff
@@ -37,6 +35,9 @@ yum -y install \
 	python-virtualenv \
 	ansible
 
+rpm -iU --force https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
```
Is 'force' needed here? I realize that these are ultimately "scratch" machines but its a bit of a "code smell"

raghavendra-talur replied:
Yes, was using the --force option to get around the rpm/yum error "does not update installed package". Replaced with an explicit check and call install only when not installed.
